package.json

```diff
@@ -49,8 +49,8 @@
     "tslint": "5.9.1",
     "typescript": "2.8.3",
     "webpack": "4.12.0",
-    "webpack-cli": "3.0.6",
-    "xhr-mock": "2.4.0"
```

Comment:
Do you really want to update this dep to a preview version? Not sure it's a good idea as we don't need this latest version?

Reply:
You are right. I've rolled back xhr-mock.